src/components/RoomView/roomView.css

```diff
@@ -68,6 +68,70 @@ body {
   margin: 15% 10% 0 0;
   padding: 10px;
   border-radius: 6px;
+  background-color: gainsboro;}
+  .MessageInput{
+    display: flex;
+    justify-content: center;
+    margin-top: 2%;
+  }
+  .setupGamePopup{      background-color: #dec160;
+    position: absolute;
+    width: 20%;
+    height: 10%;
+    padding: 3%;
+    display: flex;
+    margin: 5% auto;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    right: 50;
+    align-items: center;
+    border-radius: 5px;
+    z-index: 0;
+    justify-content: center;}
+    .setupGamePopup .close-btn{position: absolute;top:15px;right:15px;}
+    .loser{
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, 50%);
+      position: absolute;
+      z-index: 1;
+      width: 50vw;
+      height: 30vh;
+      background-color: black;
+      color: #dec160;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: 2% 5%;
+    }
+    .winner{top: 50%;
```

Comment:
Not sure exactly where the problem lays but this displays in a weird position for me. additionally, i'm not sure if we want this to be a huge popup over the board - because as soon as you win or lose it may just cover the screen and you can't tell what happened.
eg) i clicked the black card and suddenly i just have a big message saying i lost on the screen and i have no idea what happened, only once i close that can i see the board, and see that the black card was revealed.

however this is just css and we can adjust later. i don't want to hold up the logic for this PR with just styling stuff.